src/index.js

```diff
@@ -421,4 +422,12 @@ export class HTMLText extends Sprite
         this._resolution = value;
         this.dirty = true;
     }
+
+    sanitiseText(text){
+        // Sanitise text - replace <br> with <br/>, &nbsp; with &#160;
+        // See discussion here: https://www.sitepoint.com/community/t/xhtml-1-0-transitional-xml-parsing-error-entity-nbsp-not-defined/3392/3
+        text = text.replace(new RegExp('<br>'.replace(/[-\/\\^$*+?.()|[\]{}]/g, '\\$&'), 'g'), '<br/>');
```

Comment:
This regex seems a little verbose for what this is. Could you do this simpler instead:

```
/<br>/gi
```

We also may want to handle other non terminating single tags like `<hr>` as well as attributes like style. But for now this seems simple enough.